src/engine/renderer/tr_shader.cpp

```diff
@@ -4084,7 +4084,7 @@ static bool ParseShader( const char *_text )
 
 			tr.sunLight[ 2 ] = atof( token );
 
-			VectorNormalize( tr.sunLight );
+			VectorNormalizeFast( tr.sunLight );
```

Comment:
I don't see any point to using the fast version for something that's done very infrequently and only at loading time.